cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -52,6 +52,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == Known issues
 
+* KubeMacPool is disabled in {CNVProductName} {CNVVersion}. This means that the secondary
+interfaces of Pods and virtual machines obtain a randomly generated MAC address, rather
+than a unique one from a pool. In rare occurences, it is possible that conflicts in
```

Comment:
- s/In rare occurrences, it is possible that conflicts in assigned MAC addresses can occur./Rarely, assigned MAC addresses might conflict.
(or something of that nature)